[2024-09-11T03:56:08.919+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T03:56:08.937+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgers_hooks_v04.postgres_to_s3 scheduled__2022-09-21T00:00:00+00:00 [queued]>
[2024-09-11T03:56:08.947+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgers_hooks_v04.postgres_to_s3 scheduled__2022-09-21T00:00:00+00:00 [queued]>
[2024-09-11T03:56:08.948+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T03:56:09.031+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): postgres_to_s3> on 2022-09-21 00:00:00+00:00
[2024-09-11T03:56:09.042+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=28537) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T03:56:09.043+0000] {standard_task_runner.py:63} INFO - Started process 28542 to run task
[2024-09-11T03:56:09.042+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgers_hooks_v04', 'postgres_to_s3', 'scheduled__2022-09-21T00:00:00+00:00', '--job-id', '4392', '--raw', '--subdir', 'DAGS_FOLDER/17_dag_with_postgress_hook.py', '--cfg-path', '/tmp/tmpdsr054wl']
[2024-09-11T03:56:09.044+0000] {standard_task_runner.py:91} INFO - Job 4392: Subtask postgres_to_s3
[2024-09-11T03:56:09.097+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgers_hooks_v04.postgres_to_s3 scheduled__2022-09-21T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T03:56:09.203+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='dag_with_postgers_hooks_v04' AIRFLOW_CTX_TASK_ID='postgres_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2022-09-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-09-21T00:00:00+00:00'
[2024-09-11T03:56:09.204+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T03:56:09.224+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:212 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2024-09-11T03:56:09.233+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-09-11T03:56:09.246+0000] {17_dag_with_postgress_hook.py:38} INFO - Saved orders data in text file: dags/get_orders_20220921.txt
[2024-09-11T03:56:09.255+0000] {base.py:84} INFO - Using connection ID 'minio_s3_conn' for task execution.
[2024-09-11T03:56:09.256+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio_s3_conn', conn_type='aws') credentials retrieved from login and password.
[2024-09-11T03:56:09.649+0000] {17_dag_with_postgress_hook.py:48} INFO - Orders file /tmp/tmp4g3gxdgg20220921 has been pushed to S3!
[2024-09-11T03:56:09.649+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-11T03:56:09.650+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T03:56:09.680+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgers_hooks_v04, task_id=postgres_to_s3, run_id=scheduled__2022-09-21T00:00:00+00:00, execution_date=20220921T000000, start_date=20240911T035608, end_date=20240911T035609
[2024-09-11T03:56:09.738+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T03:56:09.754+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-11T03:56:09.759+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
